.env.example

```diff
@@ -9,6 +9,7 @@ VUE_APP_DASHBOARD_URL="https://dev.business.bcregistry.gov.bc.ca/"
 VUE_APP_ENTITY_SELECTOR_URL="https://entity-selection-dev.apps.silver.devops.gov.bc.ca/"
 VUE_APP_PAYMENT_PORTAL_URL="https://dev.account.bcregistry.gov.bc.ca/makepayment/"
 VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
+VUE_APP_SOCIETIES_ONLINE_HOME_URL="https://dev.bcregistry.ca/societies/"
```

Comment:
Also add this to devops/vaults.env so the CI action will populate it from 1Password.

Reply:
(This file is a sample file for developers to copy into `.env`, and is also the backup list of keys for PR temporary URLs if the cloudbuild script doesn't provide it.)